### PR TITLE
Feature/presentation details 2

### PR DESCRIPTION
**Pull request by Jsyro**
Table layout, no predicate support yet. Relevant UX mockup. https://xd.adobe.com/view/20ae96da-b8b6-4dca-8d17-da1c3403613f-75fd/screen/ad2e839c-1b83-4500-88f5-ff5c5e4fda01/

Show requested attributes (groups, single, and self-attested) in one table. Checkmark to indicate the data was verified against something, info icon brings up modal with more details on the restrictions. 

image

restrictions infoModal
image

### CODE REVIEW

**Review by usingtechnology — APPROVED** [review]
Conditionally approve this. will let others review, we shouldn't be leaving @ts-expect-error comments but needed them so I could build and review.

**Comment by Jsyro** [comment]
> Conditionally approve this. will let others review, we shouldn't be leaving @ts-expect-error comments but needed them so I could build and review.

Thanks for catching the linting, forgot again, we should look at making that a GH action.

Unsure about the `@ts-expect-error` usage, doing some research now.

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,4 +5,16 @@
...
+      "previewLimit": 50,
+      "server": "localhost",
+      "port": 5432,
+      "driver": "PostgreSQL",
+      "name": "docker-traction",
+      "username": "postgres",
+      "password": "postgresPass",
+      "database": "traction"
+    }
+  ],
```
Comment (loneil) [inline .vscode/settings.json:19]: Don't think this was to be comitted?

File: services/tenant-ui/frontend/src/components/verifier/VerifiedPresentationData.vue [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  return Object.fromEntries(
+    Object.entries(
+      props.presentation.acapy.presentation_exchange.presentation_request
+        .requested_attributes
+    ).filter(([key, ra]) => {
+      // @ts-expect-error types have not been defined for this object
+      return 'name' in ra && !('restrictions' in ra);
+    })
+  );
+};
```
Comment (loneil) [inline services/tenant-ui/frontend/src/components/verifier/VerifiedPresentationData.vue:149]: There's probably a bunch of utility-function-ing that can be done here to reduce it, but I'd be inclined to leave it until we done have as much in-flight or proposed changes still to Presentation UX.

File: .vscode/settings.json (hunk 1 above)
Comment (Jsyro) [reply .vscode/settings.json:19]: It's in the settings file and it's still valid for other users. If we don't want this in the repo then we should un-version the file.

I can remove it if desired.

File: services/tenant-ui/frontend/src/components/verifier/VerifiedPresentationData.vue (hunk 2 above)
Comment (Jsyro) [reply services/tenant-ui/frontend/src/components/verifier/VerifiedPresentationData.vue:149]: Yea, i know the JS here is rudimentary, but i'm also confident that any developer can understand this code. Let me know if you want me to make any changes here now.

File: .vscode/settings.json (hunk 1 above)
Comment (loneil) [reply .vscode/settings.json:19]: I'd clear it if it's dev machine specific tooling

**Review by loneil — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Jsyro) [reply .vscode/settings.json:19]: added to ignore, but this change was already in an earlier commit.

**Pull request merged**